applications/vanilla/settings/stub.php

```diff
@@ -27,58 +20,81 @@
    <li>Bookmark a discussion (click the star) to get notifications for new comments. You can edit notification settings from your profile.</li>
 </ul>
 Go ahead and edit or delete this discussion, then spread the word to get this place cooking. Cheers!";
-$CommentBody = "This is the first comment on your site and it&rsquo;s an important one.
+   $Discussion = array(
+      'Name' => T('StubDiscussionTitle', "BAM! You&rsquo;ve got a sweet forum"),
```

Comment:
There is no way for this translation to take effect since the content is added on install.